packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -47,6 +48,19 @@ export default class Sayt extends LitElement {
    * The minimum length of the search term required before a SAYT request will be made with it.
    */
   @property({ type: Number, reflect: true }) minSearchLength = 3;
+  /**
+   * The debounce delay in millilseconds.
+   */
+  @property({ type: Number, reflect: true }) debounce = 300;
+  /**
+   * The debouncedRequestSaytProducts method that will wrap the requestSaytProducts method in a debounce.
+   */
+  debouncedRequestSaytProducts: RequestSaytType;
```

Comment:
@jmbtutor You mentioned in one of your reviews that this shouldn't be necessary, do you have any further direction as to how to set these debounced methods?